models/EventModel.ts

```diff
@@ -73,6 +73,9 @@ export class EventModel extends BaseEntity {
   @Column('varchar', { nullable: true })
   googleCalendarEvent: Uuid;
 
+  @Column('varchar', { nullable: true })
+  food: string;
```

Comment:
Extreme nit but I think `foodItems` is a better name, `food` feels a little informal